tests/acceptance/lib/files/system_owned/system_owned_from_augments.cf

```diff
@@ -0,0 +1,45 @@
+# Tests that system_owned body takes value from augements when present
+
+body common control
+{
+      inputs => { "../../../default.cf.sub" };
+      bundlesequence => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  vars:
+    windows::
+      "tmp" string => "C:\temp";
+    !windows::
+      "tmp" string => "/tmp"; # /var/tmp?
+}
+
+bundle agent test
+{
```

Comment:
Can you please add 
```
meta:
  "description" -> { "TICKET" }
    string => "blah blah"; 
```

Reply:
added, thanks!

Reply:
oops, wrong syntax `s/"string"/string/` is needed here. Doing that and see how travis reacts.